documentation/documentation/docs/evaluators/python_evaluator.md

```diff
@@ -0,0 +1,11 @@
+# Python evaluator
+## General usage
+This evaluator is responsible for running and executing tests on a student's Python code. juuw
+
+## Structure
+When submitting the project a teacher can add a requirments manifest `filename of manifest`, this way only the packages in the requirments file are installable on the evaluator.
```

Comment:
requirments -> requirements (x2)
I would say usable instead of installable, because they will be installed on the evaluator

Reply:
https://github.com/SELab-2/UGent-3/pull/250/commits/c2af2565e6508f3e8caa588f821307480a79c078